src/less/components.less

```diff
@@ -29,6 +29,5 @@
 @import 'components/dropdown-select/index';
 @import 'components/file/index';
 @import 'components/form/index';
-@import 'components/radio/index';
```

Comment:
* `src/less/components/radio/index.less` still exists.
* Will remove this file when `docs/styles/spec/radio-spec.less` is refactored in SURF-1950.